game/theater/theatergroundobject.py

```diff
@@ -44,56 +51,148 @@
 }
 
 
-GroupT = TypeVar("GroupT", ShipGroup, VehicleGroup)
+@dataclass
+class GroundUnit:
+    # Units can be everything.. Static, Vehicle, Ship.
+    id: int
```

Comment:
It is used as unique_id generated from `game.next_unit_id()` It will be used for the later dcs group and unit name which have to be unique. Same behaviour as previoulsy just more cleaner. It has not a comment about it because it was never changed.